### PR TITLE
Diamond and Platinum customers exempt from domain validation

### PR DESCRIPTION
**Pull request by rachelwhitton**
## Summary
Adds a new release note ["Diamond and Platinum customers now exempt from domain verification enforcement"](https://pr-9197-documentation.appa.pantheon.site/release-notes/2024/08/domain-verification)

Updates to the following pages: 
* **[Relaunch Existing Pantheon Site](https://pr-9197-documentation.appa.pantheon.site/relaunch)** - Restore older iteration of this procedure in a separate tab (All others vs Diamond and Platinum) 
* **[Domains on Pantheon > Custom Domains](https://pr-9197-documentation.appa.pantheon.site/guides/domains/custom-domains)** - Update FAQ, update alert 
* **[Launch Essentials > Connect a Domain Name](https://pr-9197-documentation.appa.pantheon.site/guides/launch/domains)** - update alert 

## Deferred 
To reduce complexity and scope of this PR, I'm deferring the revision of our [domain hijacking doc](https://docs.pantheon.io/guides/domains/domain-hijacking) page to be handled in a separate issue which will define Pantheon's platform mitigations:  
* domain validation enforcement
* domain locking (not sure that's what we'd call it?)
  * "if you are trying to add a sub domain in an org that does not own the apex of that subdomain, you won't be able to add it"

### CODE REVIEW

File: source/releasenotes/2024-08-29-domain-verification.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: "Diamond and Platinum customers now exempt from domain verification enforcement"
+published_date: "2024-08-29"
+categories: [security, infrastructure]
```
Comment (rachelwhitton) [inline source/releasenotes/2024-08-29-domain-verification.md:4]: Not sure about these categories, but it's what [the last related release note used ](https://docs.pantheon.io/release-notes/2024/07/domain-validation)

File: source/releasenotes/2024-08-29-domain-verification.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+---
+Diamond and Platinum customers are now automatically exempt from domain verification enforcement on Pantheon when connecting custom domains in the site dashboard. Previously, an exemption had to be requested by contacting support.
+
+We are making this change to reduce the pain of adding domains for our trusted Diamond and Platinum customers.
```
Comment (jazzsequence) [inline source/releasenotes/2024-08-29-domain-verification.md:8]: > To reduce the pain of adding domains for our trusted Diamond and Platinum customers, Diamond and Platinum customers (orgs?) are now automaticallyl exempt from domain verification enforcement. Previously, when connecting custom domains in the Pantheon site dashboard, an exemption had to be requested by contacting support. 

Better maybe? I don't like saying "Diamond and Platinum customers" twice in a row, but I think we're talking about Diamond/Platinum orgs right? So maybe the second one could be organizations? I feel like this rephrasing flows a little bit better.

File: source/content/guides/domains/05-custom-domains.md [hunk 3]
```diff
@@ -100,9 +100,7 @@ For any domain that has been added that is pending verification, clicking on the
 You can get a high-level status view for all custom domains connected to a given environment via Terminus using the [`https:info` command](/terminus/commands/https-info). Domains that are pending verification will have the "Verification Pending" status returned as part of the Terminus `https:info` command.
 
 ### Can I opt-out of Domain Verification?
-Yes, a professional workspace can opt-out from domain verifications across all sites in their organization but only by request. Please [contact support](/guides/support/contact-support/) to request exemption from domain verification, and once granted - you will see an option to skip domain verification when connecting domains to sites in your workspace.
-
-Exemption is only available at the workspace level. For example, you cannot request exemption for just one site in your workspace, or an individual custom domain such as `example.com`.
+Sites associated with Diamond and Platinum workspaces are automatically exempt from domain verification enforcement. For those exempt, there is an option to skip domain verification when connecting domains to sites in your workspace.
```
Comment (christianyates) [inline source/content/guides/domains/05-custom-domains.md:103]: This is somewhat unclear. Is it automatic or do some customers need to take action to skip it?

Comment (rachelwhitton) [reply source/content/guides/domains/05-custom-domains.md:103]: How about replacing the last sentence with this: 

> Exemption enables the option to skip domain verification when connecting custom domains in the site dashboard.

File: source/content/guides/domains/05-custom-domains.md [hunk 4]
```diff
@@ -100,9 +100,7 @@ For any domain that has been added that is pending verification, clicking on the
 You can get a high-level status view for all custom domains connected to a given environment via Terminus using the [`https:info` command](/terminus/commands/https-info). Domains that are pending verification will have the "Verification Pending" status returned as part of the Terminus `https:info` command.
 
 ### Can I opt-out of Domain Verification?
```
Comment (rachelwhitton) [inline source/content/guides/domains/05-custom-domains.md:102]: Todo: Alll other account tiers for professional workspaces can request exemption by reaching out to support. still only the workspace level, no exemptions for a user or single site

**Comment by IngridKwok** [comment]
@rachelwhitton  I noticed that the casing for 'domain verification' is inconsistent. Since it’s not a product name or proper noun, I recommend using lowercase throughout.

Same for "site dashboard" - but since I noticed we've been using uppercase for other docs page, let's not change that for now.

File: source/releasenotes/2024-08-29-domain-verification.md (hunk 2 above)
Comment (jazzsequence) [reply source/releasenotes/2024-08-29-domain-verification.md:8]: ❤️ Love it. :shipit:

File: source/content/guides/domains/05-custom-domains.md (hunk 3 above)
Comment (christianyates) [reply source/content/guides/domains/05-custom-domains.md:103]: that works

**Pull request merged**